tasks/install-additional-software.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Install Applications from Homebrew
+  homebrew:
+    name: '{{ item }}'
+    options: force
+  loop:
```

Comment:
Haven't sense. Because you install one app.

Reply:
but there is a ready cycle in case something else appears